docs/products/opensearch/concepts/access_control.rst

```diff
@@ -1,23 +1,41 @@
 Access control
 ==============
 
-Aiven for OpenSearch® supports index-level access control lists (ACL) to control permissions. This approach allows you to limit the operations that are available to specific connections and to restrict access to certain data sets, which improves the security of your data.
+Aiven for OpenSearch® supports index-level access control lists (ACL) to control permissions.
+This approach allows you to limit the operations that are available to specific service users and
+restrict access to certain data sets.
+
+.. note::
+   When access control is initially enabled for the service, a set of rules that allows
+   full unlimited access to existing service users is created automatically.
+
+Rules are defined separately for each user as ``pattern/permission`` combinations. The ``pattern`` defines the indices that the permission applies to.
+Patterns are glob-style, where ``*`` matches any number of characters (including none) and ``?`` matches any character.
 
 You can grant the following permissions:
 
-* ``deny``: no access
-* ``admin``: full access to APIs and documents 
+* ``deny``: explicitly deny access
+* ``admin``: unlimited access to the index
 * ``readwrite``: full access to documents
 * ``read``: allow only searching and retrieving documents
 * ``write``: allow updating, adding, and deleting documents
 
-  .. note::
-     Write permission allows the service user to create new indexes that match the pattern, but it does not allow deletion of those indexes.
+The permission also implies which *index APIs* the service user can access:
+
+* ``deny``: no access
+* ``admin``: no restrictions
+* ``readwrite``: ``_search``, ``_mget``, ``_bulk``, ``_mapping``, ``_update_by_query``, ``_delete_by_query``
+* ``read``: ``_search``, ``_mget``
+* ``write``: ``_bulk``, ``_mapping``, ``_update_by_query``, ``_delete_by_query``
 
+.. note::
+   When no rules match, access is **implicitly denied**.
 
-Rules are defined separately for each user as ``pattern/permission`` combinations. The ``pattern`` defines the indexes that the permission applies to. Patterns are glob-style, where ``*`` matches any number of characters and ``?`` matches any character. 
+.. note::
+   ``write`` permission allows creating indices that match the rule's index pattern but does now allow deleting them.
+   Indices can only be deleted when a matching ``admin`` permission rule exists.
 
-When multiple rules match, they are applied in the order listed above. If no rules match, access is denied.
+When multiple rules match, they are applied in the order listed above regardless of the rule order (see the example below).
```

Comment:
What is the order? should we move from dots to numbering to show the order? also i would mention before the ordered list "the following is ordered by importance"